app/mailers/mentees_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class MenteesMailer < ApplicationMailer
+	def register(user)
```

Comment:
Not being used as for now. Just added it to be used later if a separate mailer for mentees is intended. Let me know if it's to be removed.